README.md

```diff
@@ -185,7 +185,7 @@ Run frontend tests:
 
 ### Integration Tests
 
-Integration tests are run in a headless Chrome driven by [protractor](http://www.protractortest.org/#/).  Requirements include Chrome, a working cluster, kubectl, and bridge itself (see building above).
+Integration tests are run in a headless browser driven by [protractor](http://www.protractortest.org/#/).  Requirements include Chrome or Firefox, a working cluster, kubectl, and bridge itself (see building above). By default, it will look for Chrome in the system and use it, but if you want to use Firefox instead, set `E2E_BROWSER` environment variable in your shell with the value "firefox."
```

Comment:
Better to add line breaks. It makes comments easier in PR reviews.

I'd use backticks in the value avoid any ambiguity with the period and quotation marks.

```suggestion
Integration tests are run in a headless browser driven by [protractor](http://www.protractortest.org/#/).  Requirements include Chrome or Firefox, a working cluster, kubectl, and bridge itself (see building above). By default, it will look for Chrome in the system and use it, but if you want to use Firefox instead, set `E2E_BROWSER` environment variable in your shell with the value `firefox`.
```

Reply:
Sure, sounds good. Will be in the next push, along with the new env var name.